satpy/readers/fci_l1c_nc.py

```diff
@@ -267,6 +269,9 @@ def _get_dataset_measurand(self, key, info=None):
             res.attrs.pop("scale_factor")
             res.attrs.pop("warm_scale_factor")
 
+        if key['calibration'] == 'radiance':
```

Comment:
Would it make sense to do this in `calibrate_counts_to_rad`?

Reply:
good point, done :)